src/App.js

```diff
@@ -1,68 +1,89 @@
   const [toggle, setToggle] = useState(true);
+  const [totalConsumption, setTotalConsumption] = useState({});
   const oldInputLength = initialInputData.length;
   const actualDate = new Date();
-  const formattedActualDate = actualDate.toLocaleDateString('en-GB', {year: 'numeric', month: '2-digit', day: '2-digit'})
-  const lastInputValue = inputs[0].value
+  const formattedActualDate = actualDate.toLocaleDateString('en-GB', {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  });
+
+  useEffect(() => {
+    function handleCalc() {
+      const total = inputs.map(input => input.increase).reduce((a, b) => a + b, 0);
+      const averageIncrease = total / inputs.length;
+      const averageIncreaseRounded = Math.round(averageIncrease);
+      setTotalConsumption({total, averageIncreaseRounded});
+    }
+    handleCalc();
+  }, [inputs]);
+
+  const indexLastInput = inputs.length;
+  const totalDate = actualDate - inputs[indexLastInput - 1].date;
+  console.log(formattedActualDate);
+  console.log(actualDate);
+  console.log(totalDate);
 
   function updateInput(inputDataValue) {
     const newInput = {
       id: nanoid(),
       date: formattedActualDate,
       value: Number(inputDataValue),
-  };
+      increase: inputDataValue - inputs[0].value,
+    };
 
-  if (newInput.value >= inputs[0].value)
-      {setInputs([newInput, ...inputs])
-    if (oldInputLength !== inputs.length)
-      {setShowMessage(messages[1].text)}
-    else
-      {setShowMessage(messages[2].text)}
+    if (newInput.value >= inputs[0].value) {
+      setInputs([newInput, ...inputs]);
+      if (oldInputLength !== inputs.length) {
+        setShowMessage(messages[1].text);
+      } else {
+        setShowMessage(messages[2].text);
+      }
+    } else {
+      setShowMessage('please input > or = ' + inputs[0].value);
     }
-  else 
-    {setShowMessage("please input > or = " + inputs[0].value)}
+  }
 
-}
   return (
     <>
-    <Header showMessage={showMessage} />
-    {(toggle === false) && <ReturnButton onReturn={() => setToggle(!toggle)} />}
-    <MainHeading>Energy-Budget-App</MainHeading>
-    {(toggle === true) &&
-    <CategoryButton lastInputValue={lastInputValue} onSelect={() => setToggle(!toggle)} />}
-    {(toggle === false) &&
-    <MainContainer>
-      <section>
-        <h2>{formattedActualDate}</h2>
-        <div>
-          <article>total entries: {inputs.length}</article>
-          <article>total consumption: 7000</article>
-          <article>daily average: 700</article>
-        </div>
-      </section>
-      <ul>
-      {inputs.map(({ date, value, id, increase}) => (
-          <li key={id}>
-            {date} - {value} watt/h - increase: {increase}
-          </li>
-      ))}
-      </ul>
-      <InputDataDialog updateInput={updateInput} />
-    </MainContainer>}
+      <Header showMessage={showMessage} />
+      {toggle === false && <ReturnButton onReturn={() => setToggle(!toggle)} />}
```

Comment:
There is no need to write `toggle === false`. It is more common to write it like this:

```suggestion
      {!toggle && <ReturnButton onReturn={() => setToggle(!toggle)} />}
```